utils/installer/install.sh

```diff
@@ -5,198 +5,193 @@ set -o nounset # error when referencing undefined variable
+  [ -f "/etc/gentoo-release" ] && installpipongentoo
+  [ "$(uname -s | cut -c 1-10)" = "MINGW64_NT" ] && echo "Windows not currently supported"
 }
 
 installpynvim() {
-	echo "Installing pynvim..."
-	if [ -f "/etc/gentoo-release" ]; then
-		echo "Installing using Portage"
-		sudo emerge -avn dev-python/pynvim
-	else
-		pip3 install pynvim --user
-	fi
+  echo "Installing pynvim..."
+  if [ -f "/etc/gentoo-release" ]; then
+    echo "Installing using Portage"
+    sudo emerge -avn dev-python/pynvim
+  else
+    pip3 install pynvim --user
+  fi
 }
 
 installpacker() {
-	git clone https://github.com/wbthomason/packer.nvim ~/.local/share/lunarvim/site/pack/packer/start/packer.nvim
+  git clone https://github.com/wbthomason/packer.nvim ~/.local/share/lunarvim/site/pack/packer/start/packer.nvim
 }
 
 cloneconfig() {
-	echo "Cloning LunarVim configuration"
-	mkdir -p ~/.local/share/lunarvim
-	case "$@" in
-
-	*--testing*)
-		cp -r "$(pwd)" ~/.local/share/lunarvim/lvim
-		;;
-	*)
-		git clone --branch "$LVBRANCH" https://github.com/ChristianChiarulli/lunarvim.git ~/.local/share/lunarvim/lvim
-		;;
-	esac
-	mkdir -p "$HOME/.config/lvim"
-	sudo cp "$HOME/.local/share/lunarvim/lvim/utils/bin/lvim" "/usr/local/bin"
-	cp "$HOME/.local/share/lunarvim/lvim/utils/installer/lv-config.example-no-ts.lua" "$HOME/.config/lvim/lv-config.lua"
-
-	nvim -u ~/.local/share/lunarvim/lvim/init.lua --cmd "set runtimepath+=~/.local/share/lunarvim/lvim" --headless \
-		+'autocmd User PackerComplete sleep 100m | qall' \
-		+PackerInstall
-
-	nvim -u ~/.local/share/lunarvim/lvim/init.lua --cmd "set runtimepath+=~/.local/share/lunarvim/lvim" --headless \
-		+'autocmd User PackerComplete sleep 100m | qall' \
-		+PackerSync
-
-	printf "\nCompile Complete\n"
-
-	if [ -e "$HOME/.local/share/lunarvim/lvim/init.lua" ]; then
-		echo 'lv-config already present'
-	else
-		cp "$HOME/.local/share/lunarvim/lvim/utils/installer/lv-config.example.lua" "$HOME/.config/lvim/lv-config.lua"
-	fi
-
+  echo "Cloning LunarVim configuration"
+  mkdir -p ~/.local/share/lunarvim
+  case "$@" in
+
+    *--testing*)
+      cp -r "$(pwd)" ~/.local/share/lunarvim/lvim
+      ;;
+    *)
+      git clone --branch "$LVBRANCH" https://github.com/ChristianChiarulli/lunarvim.git ~/.local/share/lunarvim/lvim
+      ;;
+  esac
+  mkdir -p "$HOME/.config/lvim"
+  sudo cp "$HOME/.local/share/lunarvim/lvim/utils/bin/lvim" "/usr/local/bin"
+  cp "$HOME/.local/share/lunarvim/lvim/utils/installer/lv-config.example-no-ts.lua" "$HOME/.config/lvim/lv-config.lua"
+
+  nvim -u ~/.local/share/lunarvim/lvim/init.lua --cmd "set runtimepath+=~/.local/share/lunarvim/lvim" --headless \
+    +'autocmd User PackerComplete sleep 100m | qall' \
+    +PackerInstall
+
+  nvim -u ~/.local/share/lunarvim/lvim/init.lua --cmd "set runtimepath+=~/.local/share/lunarvim/lvim" --headless \
+    +'autocmd User PackerComplete sleep 100m | qall' \
+    +PackerSync
+
+  printf "\nCompile Complete\n"
+  rm "$HOME/.config/lvim/lv-config.lua"
```

Comment:
That's a sneaky change, FYI you can remove the rm command, `cp` will overwrite the file

Reply:
Otherwise, what's blocking us from merging this ?

Reply:
I have no idea, I thought it was pretty straight-forward to merge this tbh. And it's blocking the other 2 PRs because I didn't want to format them weirdly 😅